From b11d83df189670defe4a29c624f2930351c13df2 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Wed, 4 Mar 2015 13:56:36 -0500
Subject: [PATCH] do not run ldconfig

Let the distro/user deal with ldconfig updating.  Running it blindly like
this breaks DESTDIR installs as `ldconfig` only operates on system paths.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
--- a/Makefile
+++ b/Makefile
@@ -127,7 +127,6 @@ install-libreg:
 	$(NQ) '  INSTALL  libreg'
 	$(Q)mkdir -p $(DESTDIR)/$(LIBDIR)
 	$(Q)$(INSTALL) -m 644 $(LIBREG) $(DESTDIR)/$(LIBDIR)/
-	$(Q)ldconfig
 
 %.o: %.c regdb.h $(LIBREG)
 	$(NQ) '  CC  ' $@
